.azure-pipelines/test-docker-sonic-vs-template.yml

```diff
@@ -120,6 +120,7 @@ jobs:
       sudo /sbin/ip link add Vrf1 type vrf table 1001 || { echo 'vrf command failed' ; exit 1; }
       sudo /sbin/ip link del Vrf1 type vrf table 1001
       pushd tests
+      mkdir -p logs
```

Comment:
What is this change? Please remove

Reply:
removed already.
the tests were passing on my buidl enviorment but failing in the pipeline. This was due to some difference in SAI searialize API. I have removed these extra log generation parameters.